src/app/lib/config/services/news.js

```diff
@@ -6,6 +6,7 @@ import * as fonts from '@bbc/psammead-styles/fonts';
 const news = {
   articleAuthor: `https://www.facebook.com/bbcnews`,
   brandName: 'BBC News',
+  product: 'BBC News',
```

Comment:
Added this because Storybook fails on the news service without it. I don't know if it is required

Reply:
I think the issue is Brand container is now expecting a key called `product` on context, where before it was using `brandName` (the value on the previous line)

You may also have to make this change to the other languages (or have Brand container continue to use `brandName` from the ServiceContext, instead of this new `product` value

Reply:
Aah, yes. I thought there might be a reason why it was added to other services but not to news.